### PR TITLE
feat: Partial arm64 support

### PR DESCRIPTION
**Pull request by DRK3**
Updated scripts and the README to add support for running on arm64. Some issues still remain - these will be addressed in another commit.

Signed-off-by: Derek Trider <Derek.Trider@securekey.com>

### CODE REVIEW

File: scripts/minikube_setup.sh [hunk 1]
```diff
@@ -14,14 +14,26 @@ set -e
 : ${KUBERNETES_VERSION:=v1.21.2}
 
 OS=$( uname -s | tr '[:upper:]' '[:lower:]' )
+ARCH=$( uname -m | sed 's/x86_64/amd64/')
```
Comment (alialkhalidi) [inline scripts/minikube_setup.sh:17]:
```
UNAME=$( uname -m)
case $ARCH in
   x86_64)
     ARCH="amd64"
     ;;
   aarch64)
     ARCH="arm64"
     ;;
   *)
     ARCH="amd64"
     ;;    
esac
```

**Comment by alialkhalidi** [comment]
I suggest you use the following variable to amend the docker run flags cases:
```
export CONTAINER_CMD_RUN_OPTS="--platform linux/amd64"
```

File: scripts/minikube_setup.sh (hunk 1 above)
Comment (DRK3) [reply scripts/minikube_setup.sh:17]: > ```
> UNAME=$( uname -m)
> case $ARCH in
>    x86_64)
>      ARCH="amd64"
>      ;;
>    aarch64)
>      ARCH="arm64"
>      ;;
>    *)
>      ARCH="amd64"
>      ;;    
> esac
> ```

Thanks, I will change it to this. One small issue is that code will convert `arm64` into `amd64` due to that final case, so if the ARCH gets reported as arm64 then it'll fail. (I'll fix that when I update my PR)

Comment (DRK3) [reply scripts/minikube_setup.sh]: Updated the scripts. Note that I removed the `*)` case as that causes `arm64` to get incorrectly converted into `amd64`.

**Review by talwinder50 — APPROVED** [review]

**Review by alialkhalidi — APPROVED** [review]

**Pull request merged**